### PR TITLE
Scheduled query changes

### PR DESCRIPTION
**Pull request by susodapop**
Updates related to [#3798](https://github.com/getredash/redash/pull/3798). Also I also revised the introductory wording, updated the PNG's for more recent UI (these have been through tinyPNG), and included an emphasis that scheduled queries ❤️ alerts.

Will merge this after #3798 is merged.

### CODE REVIEW

**Comment by ranbena** [comment]
@susodapop you'll update the failure-report screenshot after #3798 lands, ya?

**Comment by susodapop** [comment]
Yessir @ranbena. Just want something in here so we're ready to roll when it merges.

File: src/pages/kb/user-guide/querying/scheduling-a-query.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ We _strongly recommend_ that you configure schedules for any queries tied to [Al
 
 When running queries with parameters on a schedule the scheduler will use the default parameter values for the query execution.
 
+## Failed Scheduled Query Reports
 
-#### Scheduled Queries with Parameters
+If your scheduled query executions fail for some reason, Redash will notify your organizations admin group by email.
```
Comment (rauchy) [inline src/pages/kb/user-guide/querying/scheduling-a-query.md]: In the original issue I did mention this should be sent out to the organization admin, but eventually I ended up sending these e-mails to the query owner.

Comment (rauchy) [inline src/pages/kb/user-guide/querying/scheduling-a-query.md]: Also, missing apostrophe in "organizations admin" 😋

Comment (rauchy) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md]: We should indicate that (a) we send out hourly aggregated reports (not immediately after an error occurs) and (b) if you ignore these e-mails, they will auto-mute themselves at the 100th occurrence of each error.

Comment (susodapop) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md]: Fixed this in 491680c136a5b541162164f5ecd5a425fb51acda

Comment (susodapop) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md]: Fixed in 491680c136a5b541162164f5ecd5a425fb51acda

**Comment by rauchy** [comment]
LGTM 👍

File: src/pages/kb/user-guide/querying/scheduling-a-query.md [hunk 2]
```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
 slug: scheduling-a-query
 ---
 
-By default, your query won't have any scheduling but it's super easy to
-adjust!
-
-In the bottom left corner, you'll see the schedule area:
+You can use scheduled query executions to keep your public dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/scheduling-a-query.md]:
```suggestion
You can use scheduled query executions to keep your dashboards updated or to power routine Alerts. By default, your queries will not have a schedule. But this is easy to adjust. In the bottom left corner of the query editor you'll see the schedule area:
```

(it's relevant for any type of dashboard)

File: src/pages/kb/user-guide/querying/scheduling-a-query.md [hunk 3]
```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
...
 
 <img src="/assets/images/docs/gitbook/query-details.png" width="100%">
 
-Clicking ' Never ' will open a picker for "every x" or at a specific time of
-day. Note: The time you set the scheduling of queries is based on the timezone
-of the computer you're using to set it (then it's translated to UTC).
+Clicking `Never` will open a picker with allowed schedule intervals. Note: The time you set the scheduling of queries is based on the timezone of the computer you're using to set it (then it's translated to UTC).
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/scheduling-a-query.md]: Is it clear that we store the UTC value?

File: src/pages/kb/user-guide/querying/scheduling-a-query.md [hunk 4]
```diff
@@ -6,23 +6,28 @@ title: Scheduling A Query
...
+{% endcallout %}
+
+## Scheduled Queries with Parameters
+
+When running queries with parameters on a schedule the scheduler will use the default parameter values for the query execution.
+
+## Failed Scheduled Query Reports
 
-#### Scheduled Queries with Parameters
+If one or many scheduled query executions fails for some reason, Redash will send an hourly report to the queries' owners detailing the failures. These emails go out regularly every hour until there are no more failures. If you ignore them, the emails will mute themselves after the hundredth (100th) occurence of each error. Importantly, these failure report emails run on a job that is independent of your query schedules. It may take up to an hour after a failed query execution before you receive an email report.
```
Comment (arikfr) [inline src/pages/kb/user-guide/querying/scheduling-a-query.md]: It's better to be more vague about the thresholds, as those might change :)

**Comment by susodapop** [comment]
Closing this one. Picking back up here https://github.com/getredash/website/pull/403

File: src/pages/kb/user-guide/querying/scheduling-a-query.md (hunk 4 above)
Comment (susodapop) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md:31]: Fixed c43fcf0

File: src/pages/kb/user-guide/querying/scheduling-a-query.md (hunk 3 above)
Comment (susodapop) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md:13]: Fixed with an explainer c43fcf0

File: src/pages/kb/user-guide/querying/scheduling-a-query.md (hunk 2 above)
Comment (susodapop) [reply src/pages/kb/user-guide/querying/scheduling-a-query.md:9]: Fixed in c43fcf0.

**Pull request closed without merging**